### PR TITLE
Updated Quickstart 

### PR DESCRIPTION
**Pull request by jasSust**
*In Quickstart guide table, changed Extreme Precipitation description from a probabilistic projection to annual days per year where precipitation exceeds 51mm.

### CODE REVIEW

File: jekyll/userguide.md [hunk 1]
```diff
@@ -33,19 +32,19 @@ We also use fire weather as an indicator of fire risk.  This indicator shows the
...
+We base our modeling of inland reverine flooding exposure on [recent advances](https://www.nature.com/articles/s41558-018-0257-z?WT.feed_name=subjects_hydrology) in combining climate models, hydrological models, local flood defense data, and high-resolution elevation maps. Coastal flooding from sea level rise or storm surges are not considered in this metric. When calculating whether a flood will occur, the framework incorporates local resilience information, such as 1 in 100 year design thresholds for dams and levees, based on a [global atlas](https://www.un-spider.org/links-and-resources/data-sources/flopros-evolving-global-database-flood-protection-standards-vrije) of flood defenses.
```
Comment (bcwaldon) [inline jekyll/userguide.md:40]: should this be `riverine`?

**Review by bcwaldon — CHANGES_REQUESTED** [review]
One typo, I think

File: jekyll/userguide.md (hunk 1 above)
Comment (bcwaldon) [reply jekyll/userguide.md:40]: @jasSust ^

Comment (jasSust) [reply jekyll/userguide.md]: Correct -- fixed

**Comment by jasSust** [comment]
*Updated this PR with a number of typo fixes to the user guide

**Pull request merged**